routers/admin.py

```diff
@@ -0,0 +1,50 @@
+from typing import Annotated
+from pydantic import BaseModel, Field
+from sqlalchemy.orm import Session
+from fastapi import APIRouter, Depends, HTTPException, Path
+from starlette import status
+from models import Todos
+from database import SessionLocal
+from .auth import get_current_user
+
+router = APIRouter(
+    prefix='/admin',
+    tags=['admin']
+)
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+db_dependency = Annotated[Session, Depends(get_db)]
+user_dependency = Annotated[dict, Depends(get_current_user)]
+
+
+@router.get("/todo", status_code=status.HTTP_200_OK)
+async def read_all(user: user_dependency, db: db_dependency):
```

Comment:
Learn to include Python type hints for output of functions:

```suggestion
async def read_all(user: user_dependency, db: db_dependency) -> list[Todo]:
```

Reply:
OK, I get it.